.travis.yml

```diff
@@ -205,3 +218,9 @@ script:
       bjam arm-tests warnings-as-errors=on ssl=$ssl crypto=$crypto variant=test_arm $toolset target-os=linux link=static testing.launcher="sudo cp -R bin rootfs/; sudo chroot rootfs";
     fi'
   - cd ..
+
+# test build for iOS, using x86_64 to avoid the need to build boost from source for arm
```

Comment:
since the ``BOOST_ROOT`` environment variable is set, the Jamfile will automatically use that boost directory, and build dependencies from it. To force a build from the "system" boost, you have to unset it. But, does brew's boost package really contain arm binaries?